yt/visualization/plot_modifications.py

```diff
@@ -211,16 +216,15 @@ def sanitize_coord_system(self, plot, coord, coord_system):
                 2D coordinates within figure object from (0,0) in lower left
                 to (1,1) in upper right.  Same as matplotlib figure coords.
         """
+        # Assure coords are either a YTArray or numpy array
```

Comment:
Thanks to @ngoldbaum for the suggestion on this numpy function of which I was unaware, which preserves YTArrays or wraps as a numpy array.